<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckItem</class>
 <widget class="QWidget" name="CheckItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_deadline {font-family: &quot;Georgia&quot;, &quot;NSimSun&quot;, &quot;Times New Roman&quot;, &quot;SimSun&quot;;}
#label_title {font-family: &quot;Segoe UI&quot;, &quot;DengXian&quot;, &quot;Calibri&quot;, &quot;Microsoft YaHei UI&quot;, &quot;Arial&quot;, &quot;SimHei&quot;;}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="button_check">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="icon">
      <iconset resource="FontAwesome.min.qrc">
       <normaloff>:/FontAwesome/svgs/regular/circle.svg</normaloff>:/FontAwesome/svgs/regular/circle.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Title 标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_deadline">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Georgia</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: grey;</string>
       </property>
       <property name="text">
        <string notr="true">1926-08-17</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_delete">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="icon">
      <iconset resource="FontAwesome.min.qrc">
       <normaloff>:/FontAwesome/svgs/solid/trash-alt.svg</normaloff>:/FontAwesome/svgs/solid/trash-alt.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="FontAwesome.min.qrc"/>
 </resources>
 <connections/>
</ui>
